internal/manifests/monolithic/configmap_test.go

```diff
@@ -222,9 +222,9 @@ distributor:
       protocols:
         grpc:
           tls:
-            client_ca_file: /var/run/ca-receiver/service-ca.crt
-            cert_file: /var/run/tls/receiver/tls.crt
-            key_file: /var/run/tls/receiver/tls.key
+            client_ca_file: /var/run/ca-receiver/grpc/service-ca.crt
```

Comment:
Shall we add a test to use the same certs for `otlp.protocols.http` ?

Reply:
Done